.github/workflows/ci-pytorch-test-full.yml

```diff
@@ -7,6 +7,15 @@ on:
   pull_request:
     branches: [master, "release/*"]
     types: [opened, reopened, ready_for_review, synchronize]  # add `ready_for_review` since draft is skipped
+    paths:
+      - "requirements/pytorch/**"
+      - "src/pytorch_lightning/**"
+      - "tests/tests_pytorch/**"
+      - "setup.cfg"  # includes pytest config
+      - ".github/workflows/ci-pytorch-test-full.yml"
+      - "requirements/lite/**"
+      - "src/lightning_lite/**"
+      - "tests/tests_lite/**"
```

Comment:
why is it included here? Changes to pytorch won't affect lite testing at all. 
We have ci-lite-full

Reply:
In the checkgroup config, it is set that changes to Lite should require that PL tests pass

Which means that a change to Lite needs to launch the PL tests. This is what this configures

Reply:
https://github.com/Lightning-AI/lightning/blob/326e5afafd35ab89545a739a347a5d8db27cec81/.github/checkgroup.yml#L169-L178